tools/src/Transforms.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs-extra';
+import minimatch from 'minimatch';
+import path from 'path';
+
+import { CopyFileOptions, CopyFileResult, StringTransform } from './Transforms.types';
+import { arrayize } from './Utils';
+
+export * from './Transforms.types';
+
+/**
+ * Transforms input string according to the given transform rules.
+ * Returns an object with the `output` and `matches`.
+ */
+export function transformString(
+  input: string,
+  transforms: StringTransform[] | null | undefined
+): string {
+  if (!transforms) {
+    return input;
+  }
+  return transforms.reduce(
+    // @ts-ignore @tsapeta: TS gets crazy on `replaceWith` being a function.
+    (acc, transform) => acc.replace(transform.find, transform.replaceWith),
```

Comment:
```suggestion
    (acc, { find, replaceWith }) => acc.replace(find, replaceWith),
```
A few characters less 🤓